src/components/Swipeable.js

```diff
@@ -0,0 +1,221 @@
+
+    let animatedCardStyles = {transform: [{translateX}, {translateY}, {rotate}]}
+    let leftOverlayStyle = {backgroundColor: leftOverlayColor, opacity: opacityLeft}
+    let rightOverlayStyle = {backgroundColor: rightOverlayColor, opacity: opacityRight}
+    let leftOverlayTextStyle = {opacity: opacityLeftText}
+    let rightOverlayTextStyle = {opacity: opacityRightText}
+
+    return (
+      <View style={[styles.container, {top: toTop + this.props.questionContainerHeight}]}>
+        <View style={styles.subjectContainer}>
+          <Animated.View
+            style={[styles.imageContainer, animatedCardStyles, swipeableSize]}
+            {...this._panResponder.panHandlers}>
+
+            <View style={swipeableSize}>
+              <SwipeSubject
+                inFront={true}
+                subject={this.props.subject}
+                subjectSizes={this.props.subjectSizes}
+                seenThisSession={this.props.seenThisSession}
+                setNextSubject={this.props.setNextSubject}
+              />
+              <Animated.View style={[styles.overlayContainer, leftOverlayStyle, imageSizeStyle]} />
+              <Animated.View style={[styles.overlayContainer, leftOverlayTextStyle, imageSizeStyle]}>
+                <StyledText additionalStyles={[styles.answerOverlayText]} text={ answers[0].label } />
+              </Animated.View>
+
+              <Animated.View style={[styles.overlayContainer, rightOverlayStyle, imageSizeStyle]} />
+              <Animated.View style={[styles.overlayContainer, rightOverlayTextStyle, imageSizeStyle]}>
+                <StyledText additionalStyles={[styles.answerOverlayText]} text={ answers[1].label } />
+              </Animated.View>
+            </View>
+          </Animated.View>
+        </View>
+      </View>
+    )
+  }
+}
+
+const styles = EStyleSheet.create({
+  container: {
+    position: 'absolute',
+    right: 0,
+    left: 0,
+    bottom: 0,
+    backgroundColor: 'transparent',
+    flex: 1,
+    paddingVertical: 10,
+    paddingHorizontal: 20,
+    alignItems: 'flex-start',
+    justifyContent: 'flex-start',
+  },
+  subjectContainer: {
+    position: 'absolute',
+    top: 0,
+    right: 0,
+    left: 0,
+    bottom: 0,
+    elevation: 0,
+    backgroundColor: 'transparent',
+    alignItems: 'center',
+    justifyContent: 'flex-start',
+  },
+  overlayContainer: {
+    borderRadius: 2,
+    backgroundColor: 'transparent',
+    position: 'absolute',
+    top: 20,
+    right: 0,
+    left: 0,
+    bottom: 0,
+    alignItems: 'center',
+    justifyContent: 'center',
+  },
+  answerOverlayText: {
+    fontSize: 30,
+    color: 'white'
+  },
+})
+
+Swipeable.propTypes = {
+  subject: React.PropTypes.shape({
+    id: React.PropTypes.string
+  }),
+  nextSubject: React.PropTypes.shape({
+    id: React.PropTypes.string
+  }),
+  subjectSizes: React.PropTypes.object,
+  seenThisSession: React.PropTypes.array,
```

Comment:
Let's be a bit more specific here, as we've been doing in PFE. Use `React.PropTypes.arrayOf(React.PropTypes.string/object/etc.)`. This is preferred as `React.PropTypes.array` can be a bit ambiguous. [Here](https://facebook.github.io/react/docs/typechecking-with-proptypes.html) is some info from the React docs.